ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -1555,7 +1555,7 @@ def arrays_for_pooling(
         )
     if array_dim == 3:
         kernel = draw(st.tuples(st.integers(1, in_shape[1])))
-    new_kernel = kernel
+        new_kernel = kernel
```

Comment:
Hello @mtnvdsk, good catch! can you please create a separate PR for this fix? Thanks.